.github/workflows/OSX.yml

```diff
@@ -75,7 +75,6 @@ jobs:
     # Builds binaries for osx_arm64 and osx_amd64
     name: OSX Release
     runs-on: macos-latest
-    needs: xcode-debug
```

Comment:
is this on purpose?

Reply:
This was on purpose, but this PR has no reason to be here now that https://github.com/duckdb/duckdb/pull/8679 is done.